app/domain/RequestAuthoritiesCsv.scala

```diff
@@ -18,10 +18,8 @@ package domain
 
 import play.api.libs.json.{Json, OFormat}
 
-case class RequestAuthoritiesCsv(requestingEori: String)
+case class RequestAuthoritiesCsv(requestingEori: String, alternativeEORI: Option[String])
```

Comment:
Could you please rename to alternateEORI to make it consistent with the API and the backend